### PR TITLE
Modify assert in code

### PR DESCRIPTION
**Pull request by lukyanoffpashok**

### CODE REVIEW

File: aibolit/patterns/assert_in_code/assert_in_code.py [hunk 1]
```diff
@@ -1,35 +1,14 @@
...
 
 
-class AssertInCode(object):
-    def value(self, filename: str) -> List[LineNumber]:
-        tree = build_ast(filename)
-        return self.__traverse_node(tree)
+class AssertInCode:
+    def value(self, filename: str) -> List[int]:
+        tree = AST(build_ast(filename))
+        lines = []
```
Comment (aravij) [inline aibolit/patterns/assert_in_code/assert_in_code.py:9]:
```suggestion
        lines: List[int] = []
```

**Comment by aravij** [comment]
@acheshkov please merge)

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#456` is not assigned, can't get performer

**Pull request merged**